masterlist.yaml

```diff
@@ -25181,3 +25198,16 @@ plugins:
       - <<: *useOnlyOneX
         condition: 'many("(CFR-CloaksForRequiem|Requiem - Cloaks( of Skyrim| of Skyrim Hard - Times| of Skyrim Patch| WICCloaks CCFR USKP)?)\.esp")'
         subs: [ 'Requiem patch for Cloaks of Skyrim' ]
+  - name: 'Requiem - Immersive Jewelry( BTC| ISC)?\.esp'
```

Comment:
Both `Requiem - Immersive Jewelry( (BTC|ISC))?\.esp` and `Requiem - Immersive Jewelry( BTC| ISC)?\.esp` aren't neat and the latter has one character less so I went with that.